client/my-sites/site-settings/form-general.jsx

```diff
@@ -330,6 +330,54 @@ class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	netNeutralityOption() {
+		const { fields, isRequestingSettings, translate, handleToggle, moment, handleSubmitForm, isSavingSettings } = this.props;
+
+		const today = moment(),
+			lastDay = moment( { year: 2017, month: 6, day: 12 } );
+
+		if ( today.isAfter( lastDay, 'day' ) ) {
+			return null;
```

Comment:
So, it won't be possible to turn it on _or off_ after this time? Is it possible that someone ahead of US timezone turns it on, then wants to turn it off later in the day but can't get to the setting because their day has ended?